hw/ip/spi_device/data/spi_device_testplan.hjson

```diff
@@ -146,5 +146,99 @@
       milestone: V2
       tests: []
     }
+    {
+      name: tpm_read
+      desc: '''
+            - Set TPM_CFG.TPM_MODE to 0 and set TPM_CFG.EN.
+            - Randomise other fields in TPM_CFG.
+            - Assert the tpm_csb.
```

Comment:
nit: csb is active low. should be de-assert

Reply:
'assert' is agnostic to signal level. Asserting an active low is equivalent to driving low.

Reply:
Would you prefer a more explicit description, eg "drive the csb low"

Reply:
yeah, that also makes sense. don't need to change it.